src/util.h

```diff
@@ -51,6 +51,13 @@ extern bool fLogIPs;
 extern volatile bool fReopenDebugLog;
 extern CTranslationInterface translationInterface;
 
+extern const char * const BITCOIN_CONF_FILENAME;
+extern const char * const BITCOIN_PID_FILENAME;
+
+static const bool DEFAULT_SELFSIGNED_ROOTCERTS = false;
+static const bool DEFAULT_CHOOSE_DATADIR = false;
```

Comment:
Agreed, plus if disable wallet is enabled, it'll possibly throw a tonne of unused variable warnings (assuming `-Wnounused`